### PR TITLE
[Outlook] (manifest) Note restrictions for Form's child elements

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**
Fixes https://github.com/OfficeDev/office-js-docs-pr/issues/1494.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[0d51159](https://github.com/OfficeDev/office-js-docs-pr/commits/0d51159d2d90e941de7ab10ae5d831dba0de7b54)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/manifest/desktopsettings.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-form-elements/docs/reference/manifest/desktopsettings.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/form.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-form-elements/docs/reference/manifest/form.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/phonesettings.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-form-elements/docs/reference/manifest/phonesettings.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/tabletsettings.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-form-elements/docs/reference/manifest/tabletsettings.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C20%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201912202233326148-1510%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=%2FyR07GwF2QkaT3RjjAcEF2pTJ57ewWTh8fzJcYNAt6Q%3D&st=2019-12-20T22%3A30%3A06Z&se=2020-01-20T22%3A35%3A06Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/manifest/desktopsettings.md [hunk 1]
```diff
@@ -1,25 +1,40 @@
...
+ms.date: 12/20/2019
 localization_priority: Normal
 ---
 
 # DesktopSettings element
 
 Specifies source location and control settings that apply when your mail add-in is used on a desktop computer.
 
+> [!IMPORTANT]
+> The `DesktopSettings` elements is available for classic Outlook on the web (usually connected to older versions of on-premises Exchange server) and Outlook 2013 on Windows.
```
Comment (ElizabethSamuel-MSFT) [inline docs/reference/manifest/desktopsettings.md]:
```suggestion
> The `DesktopSettings` element is available for classic Outlook on the web (usually connected to older versions of on-premises Exchange server) and Outlook 2013 on Windows.
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[77b1dc0](https://github.com/OfficeDev/office-js-docs-pr/commits/77b1dc0c362f48aa18bb2b9e7d778d80e2de8044)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/manifest/desktopsettings.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-form-elements/docs/reference/manifest/desktopsettings.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/form.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-form-elements/docs/reference/manifest/form.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/phonesettings.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-form-elements/docs/reference/manifest/phonesettings.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/tabletsettings.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-form-elements/docs/reference/manifest/tabletsettings.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C1%5C13%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202001132121461530-1510%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=8iWgWz8iR5wmRhzUKnQ92cP1vrjDGLew2W4823YGhwE%3D&st=2020-01-13T21%3A17%3A58Z&se=2020-02-13T21%3A22%3A58Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[5897f53](https://github.com/OfficeDev/office-js-docs-pr/commits/5897f53c99ab05c184a9932ab83f1219ff66e1ac)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/manifest/desktopsettings.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-form-elements/docs/reference/manifest/desktopsettings.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/form.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-form-elements/docs/reference/manifest/form.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/phonesettings.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-form-elements/docs/reference/manifest/phonesettings.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/tabletsettings.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-form-elements/docs/reference/manifest/tabletsettings.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C1%5C13%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202001132122482577-1510%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=ObQeg1wee9QpD0wVUnJG4RyK%2FwglQlAYNDb06j5uhdU%3D&st=2020-01-13T21%3A19%3A08Z&se=2020-02-13T21%3A24%3A08Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/manifest/desktopsettings.md [hunk 2]
```diff
@@ -1,25 +1,40 @@
...
-ms.date: 10/09/2018
+ms.date: 01/13/2020
 localization_priority: Normal
 ---
 
 # DesktopSettings element
 
 Specifies source location and control settings that apply when your mail add-in is used on a desktop computer.
 
+> [!IMPORTANT]
```
Comment (Rick-Kirkham) [inline docs/reference/manifest/desktopsettings.md]: Is this note meant to imply that it is not available for other Outlook hosts? If so, then it should probably say that. If not, then I don't understand the point of the note. (In other words, should the word "only" be in front of "available"?

**Comment by o365devx** [comment]
Docs Build status updates of commit _[d0145be](https://github.com/OfficeDev/office-js-docs-pr/commits/d0145beb73c4ef2c74da416e5c04ea2fd3178d2d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/manifest/desktopsettings.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-form-elements/docs/reference/manifest/desktopsettings.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/form.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-form-elements/docs/reference/manifest/form.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/phonesettings.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-form-elements/docs/reference/manifest/phonesettings.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/tabletsettings.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-form-elements/docs/reference/manifest/tabletsettings.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C1%5C14%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202001141915219756-1510%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=tpwdYZPYRkduPCuvBjSJK%2Fj80%2Fe0UVc%2BUCsd%2BFChDhU%3D&st=2020-01-14T19%3A11%3A31Z&se=2020-02-14T19%3A16%3A31Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/manifest/desktopsettings.md (hunk 2 above)
Comment (ElizabethSamuel-MSFT) [reply docs/reference/manifest/desktopsettings.md]: Thanks for pointing that out. I included the word "only" in the others but forgot it for this one. Updated now.

**Pull request merged**